### PR TITLE
2 split layout into admin and frontend

### PR DESCRIPTION
**Pull request by anhpn0712**
#2 
e gửi task chia layout thành 2 layout admin và frontend

### CODE REVIEW

**Review by khoa250200 — APPROVED** [review]

**Review by hongthamnguyen2k — APPROVED** [review]

File: app/views/layouts/frontend/mailer.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (KhoaNDAHopee) [inline app/views/layouts/frontend/mailer.html.erb]: No need to split layout for email, email have a same layout but different content, subject...

File: config/routes.rb [hunk 2]
```diff
@@ -5,4 +5,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  namespace :admin do
+    # các route cho trang admin
+    resources :admins
+  end
+  namespace :frontend do
```
Comment (KhoaNDAHopee) [inline config/routes.rb]: No need namespace for frontend

**Review by KhoaNDAHopee — CHANGES_REQUESTED** [review]
@anhpn0712 
Please, check rubocop

File: app/controllers/admin/application_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Admin::ApplicationController < ApplicationController
```
Comment (KhoaNDAHopee) [inline app/controllers/admin/application_controller.rb]: Please, rename to base_controller.rb

File: app/controllers/frontend/application_controller.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
```
Comment (KhoaNDAHopee) [inline app/controllers/frontend/application_controller.rb]: No need to split for frontend, keep all frontend controller in controllers, example: controllers/users_controllers.rb, controllers/products_controllers.rb...

**Review by KhoaNDAHopee — CHANGES_REQUESTED** [review]

**Pull request merged**